### DISCUSSION TITLE
If there's an alert switch to Alert list dashboard in a playlist

### DISCUSSION THREAD

**Discussion by csupi003** [grafana/grafana#69428] [Dashboards, Panels & Visualizations]
Hi, 

We have plenty of dashboards in a playlist and takes some time for it to cycle through and it would be great if a playlist which has Alert List in it would switch to this for x amount of time or every second dashboard would be that but only if there's an alert.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82637. Feel free to continue the discussion around this there. Thank you!
